docs/docs-beta/docs/dagster-plus/access/rbac/audit-logs.md

```diff
@@ -4,4 +4,65 @@ displayed_sidebar: "dagsterPlus"
 sidebar_position: 4
 ---
 
-# Audit logs
\ No newline at end of file
+# Audit logs
+
+The Dagster+ audit log enables Dagster+ Pro organizations to track and attribute changes to their Dagster deployment.
+
+For large organizations, tracking down when and by whom changes were made can be crucial for maintaining security and compliance. The audit log is also valuable
+ for tracking operational history, including sensor and schedule updates.
+
+This guide walks through how to access the audit log and details the interactions which are tracked in the audit log.
+
+<details>
+<summary>Prerequisites</summary>
+- A Dagster+ Pro organization
+- An [Organization Admin](/dagster-plus/access/rbac/user-roles-permissions) role in your Dagster+ organization
+</details>
+
+## View audit logs
+
+To access the audit logs:
+
+1. Click your user icon at the top right corner of the page.
+2. Click **Organization settings**.
+3. Click the **Audit log** tab.
+
+:::warning
+
+Add screenshot
+
+:::
+
+Each entry in the audit log indicates when an action was taken, the user who performed the action, the action taken, and the deployment which the action affected. To view additional details for an action, click the **Show** button.
+
+## Filter the audit log
+
+The **Filter** button near the top left of the page can be used to filter the list of logs. You can filter to a combination of user, event type, affected deployment, or time frame.
+
+## Audit log entry types
+
+| Event type                     | Description                                                                                                   | Additional details                                                        |
```

Comment:
I love this table -- how often do we add audit log events and is there a risk it will become stale?

Reply:
@OwenKephart's [auto-generating alert doc](examples/docs_beta_snippets/docs_beta_snippets/dagster-plus/deployment/alerts/generate_alerts_doc.py) makes me want to give similar treatment to everything like this. Could look into adapting it.

Reply:
That's a great question. Took a stab at making a parity unit test here which would give us some notification if the tables got out of sync: https://github.com/dagster-io/internal/pull/11224

Not sure if in practice this is more annoying than it's worth